pkgs/applications/virtualization/qemu/default.nix

```
@@ -115,8 +115,13 @@ stdenv.mkDerivation rec {
       url = "https://gitlab.com/raboof/qemu/-/commit/3fb5e8fe4434130b1167a995b2a01c077cca2cd5.patch";
       sha256 = "sha256-evzrN3i4ntc/AFG0C0rezQpQbWcnx74nXO+5DLErX8o=";
     })
+    # fix 9p on macOS host, landed in master
+    (fetchpatch {
+      url = "https://gitlab.com/raboof/qemu/-/commit/f5643914a9e8f79c606a76e6a9d7ea82a3fc3e65.patch";
```

```suggestion
      url = "https://gitlab.com/qemu/qemu/-/commit/f5643914a9e8f79c606a76e6a9d7ea82a3fc3e65.patch";
```

Since this patch has been applied upstream, we should be fetching it from upstream, not some fork.